core/gui/src/app/hub/component/workflow/detail/hub-workflow-detail.component.ts

```diff
@@ -79,20 +65,14 @@ export class HubWorkflowDetailComponent implements AfterViewInit, OnDestroy, OnI
   public pid?: number = undefined;
   userSystemEnabled = environment.userSystemEnabled;
   private currentUser?: User;
-  @ViewChild("codeEditor", { read: ViewContainerRef }) codeEditorViewRef!: ViewContainerRef;
   constructor(
     private userService: UserService,
     // list additional services in constructor so they are initialized even if no one use them directly
-    private undoRedoService: UndoRedoService,
-    private workflowPersistService: WorkflowPersistService,
-    private workflowWebsocketService: WorkflowWebsocketService,
     private workflowActionService: WorkflowActionService,
     private route: ActivatedRoute,
```

Comment:
do we need route?

Reply:
We need this to redirect to the newly cloned workflow.